pattern_import_export_xlsx/models/ir_exports.py

```diff
@@ -93,7 +93,7 @@ def _create_validators(self, main_sheet, main_sheet_length, tab_data):
                 col_letter_dst,
                 str(self.row_start_records),
                 col_letter_dst,
-                str(main_sheet_length),
+                str(max(main_sheet_length, 2)),
```

Comment:
We should put a bigger lenght.
If we generate a pattern template we want to have at least 1000 lines ready to be filled

Reply:
fixed here https://github.com/shopinvader/pattern-import-export/blob/c217a3d652f139e65cb80038b62d678f50f4f91d/pattern_import_export_xlsx/models/ir_exports.py#L31

from PR https://github.com/shopinvader/pattern-import-export/pull/33